src/test/java/core/basesyntax/service/impl/ReportCreatorImplTest.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportCreator;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+class ReportCreatorImplTest {
+    private static final String RIGHT_REPORT = "fruit,quantity"
+            + System.lineSeparator() + "banana,10";
+    private static final String EMPTY_REPORT = "fruit,quantity";
```

Comment:
no need to make everything constants. 
instantiate expected data inside test case
```suggestion
```